using-directus-as-a-baby-health-tracker/index.md

```diff
@@ -0,0 +1,403 @@
+There are a couple of fields provided by Directus, during the creation of the new Collection we've selected the following ones:
+* sort
+* status
+* date_created
+For each element of the data provided by the sensor, we want to save these values. Depending on the used sensor, you may come up with a different set of fields. In our case we've created the following important additional fields:
+
+| Field name | Interface | Comment |
+| ---- | ---- | ---- |
+| raw_payload | Code | We use this to store the JSON payload that we gather from the external API. This is more or less just for debugging and can be skipped of cause. |
+| temperature | Input, Decimal | Just the plain value of the sensor data |
+| oxygen | Input, Decimal | Just the plain value of the sensor data |
+| heart_rate | Input, Decimal | Just the plain value of the sensor data |
+| battery_level | Input, Integer | Just the plain value of the sensor data |
+| battery_status | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+| heart_status | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+| oxygen_state | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+| thermal_state | Dropdown | To evaluate the sensor data, a simple Dropdown field is used to display the status for each value. |
+#### The alerting collection (optional)
+The second Collection is only needed for the integration into OpsGenie later on, so you may skip this part if not needed in your scenario or replace/adjust it with any other structure (fields in this collection) that might become handy for your kind of reporting or integration into external tools.
+
+| Field name | Interface | Comment |
+| ---- | ---- | ---- |
+| id |  |  |
+| external_id |  |  |
+| sort |  |  |
+| user_created |  |  |
+| user_updated |  |  |
+| date_created |  |  |
+| date_updated |  |  |
+| category |  |  |
+| type |  |  |
+| severity |  |  |
+| status |  |  |
+| priority |  |  |
+| message |  |  |
+| alias |  |  |
+| details |  |  |
+| entity |  |  |
+| description |  |  |
+| neboo_data |  |  |
+| links |  |  |
+The idea behind this collection is to store the alert details directly within Directus. Whenever an alert is created within this collection, a new alert has to be created within our external alerting tool (OpsGenie). So most fields are just the the exact fields we'd like to set within OpsGenie, some are Directus-internal related and the external_id as well as the neboo_data fields are used to refer to either the alert or the regarding data set from our first data collection.
+### Flows
+
+> [!TIP] What is a Flow
+> Flows enable custom, event-driven data processing and task automation within Directus.  [Learn more about Flows](https://docs.directus.io/reference/system/flows.html) 
```

Comment:
Remove this, but be sure to replace it with a quick explainer of what the Flow is used for. You kinda jump straight in without this context.